2-mandatory/4-space-colonies.js

```diff
@@ -15,7 +15,11 @@
   
 */
 
-function getSettlers() {}
+function getSettlers(voyagers) {
+  return voyagers.filter(
+    (family) => family.startsWith("A") && family.includes("family")
```

Comment:
Again startsWith is really handy here, but there's another simple way to get the first index of word and see if it  is equal to "A"